dockerfiles/hpcc/files/roxie.yaml

```diff
@@ -0,0 +1,3 @@
+poo: "{{ $.Values.global.image.version }}"
```

Comment:
This is a test file only... should be replaced with something better once we support yaml reading